dotnet/src/Connectors/Connectors.Memory.Weaviate/WeaviateMemoryStore.cs

```diff
@@ -76,6 +86,47 @@ public WeaviateMemoryStore(string scheme, string host, int port, string? apiKey
         this._httpClient.BaseAddress = new($"{scheme}://{host}:{port}/v1/");
     }
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="WeaviateMemoryStore"/> class.
+    /// </summary>
+    /// <param name="endpoint">The Weaviate server endpoint URL.</param>
+    /// <param name="apiKey">The API key for accessing Weaviate server.</param>
+    /// <param name="logger">Optional logger instance.</param>
+    public WeaviateMemoryStore(string endpoint, string? apiKey = null, ILogger? logger = null)
```

Comment:
Do not need to change as part of this, but has there been discussion on using `Uri` instead of `string` for endpoint?

Reply:
Almost all connectors use string type for endpoint parameter and this PR follows this approach. It can be done by a separate PR to change the endpoint parameter type when/if we agreed to do so.